cluster/gce/gci/configure-helper.sh

```diff
@@ -75,16 +75,22 @@ function mount-master-pd {
   mkdir -p "${mount_point}"
   /usr/share/google/safe_format_and_mount -m "mkfs.ext4 -F" "${pd_path}" "${mount_point}" &>/var/log/master-pd-mount.log || \
     { echo "!!! master-pd mount failed, review /var/log/master-pd-mount.log !!!"; return 1; }
+
+  # NOTE: These locations are strongly coupled with locations in
+  # cluster/gce/configure-vm.sh:mount-master-pd, which specifies where files
+  # will exist on the PD for non-GCI masters.  To maintain upgradeability,
+  # these locations must not change.
```

Comment:
I wouldn't put it as "must not change". AFAIK there is no expectation that switching between os-distros for the same master/node instance (having some state) should always work.